app/controllers/state_file/intake_logins_controller.rb

```diff
@@ -91,7 +92,7 @@ def redirect_to_data_review_if_intake_authenticated
         controller = intake.controller_for_current_step
         to_path = controller.to_path_helper(
           action: controller.navigation_actions.first,
-          us_state: params[:us_state]
+          us_state: intake.state_code
```

Comment:
Pulling the state code from the intake rather than the params, so redirecting into the correct state if param is `us`